pages/getData.tsx

```diff
@@ -22,6 +22,7 @@ const dataKeyList = [
   ...LSP6DataKeys.map((key) => ({ name: key.name, key: key.key, icon: '🔑' })),
   ...LSP9DataKeys.map((key) => ({ name: key.name, key: key.key, icon: '🔒' })),
 ];
+const ADDDRESS_PERMISSIONS_PERMISSION_PREFIX = '0x4b80742de2bf82acb3630000';
```

Comment:
Please use:

```
.indexOf('AddressPermissions:Permissions:<address>')
```

as described in previous comment

Reply:
This does not work, and that's why i hardcoded the key prefix

Reply:
My bad it was `findIndex` but a quick google "find index array of objects javascript" does the trick

cf https://github.com/lukso-network/tools-erc725-inspect/pull/40/commits/50ba182b92cb73c78dd5515f197ba6fab7ddaa4f